pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -25,6 +32,5 @@ stdenv.mkDerivation rec {
     description = "Build tool that builds code quickly and reliably";
     license = stdenv.lib.licenses.asl20;
     maintainers = [ stdenv.lib.maintainers.philandstuff ];
-    platforms = [ "x86_64-linux" ];
```

Comment:
re-added. I think other platforms such as x86_64-darwin or i686-linux might work but I have no way of testing